### PR TITLE
Auto increment account

### PR DESCRIPTION
**Pull request by lvolkmann**
auto increment account id when signing up without providing an existing account id

image

image

### CODE REVIEW

**Comment by lvolkmann** [comment]
Both requests now respond with account id for the account created

File: react-backend/routes/signup.js [hunk 1]
```diff
@@ -8,26 +8,64 @@ const connection = mysql.createPool({
...
+  const newAccountQuery =
+    "insert into Account (email, password, balance) " +
+    "values (" +
+    connection.escape(email) +
+    ", AES_ENCRYPT(" +
+    connection.escape(password) +
+    ", '" +
+    config.password +
+    "'), 0);";
+
```
Comment (sfeye) [inline react-backend/routes/signup.js:40]: I can update the front end, I think I am going to just remove the accountID field.

File: react-backend/routes/signup.js [hunk 2]
```diff
@@ -8,26 +8,64 @@ const connection = mysql.createPool({
...
+        // If some error occurs, we throw an error.
+        if (error) throw error;
+
+        connection.query(getAccountIDQuery, function (error, results, fields) {
+          if (error) throw error;
+
+          console.log(results);
+          console.log(results[0]);
+          console.log(results[0]["accountID"]);
+          res.send(results[0]["accountID"].toString());
```
Comment (sfeye) [inline react-backend/routes/signup.js:58]: Can you adjust this to say something that tells the user that an account was successfully created? Whatever is passed into res.send() will pop up in a helper window.

**Review by sfeye — APPROVED** [review]

File: react-backend/routes/signup.js (hunk 2 above)
Comment (sfeye) [reply react-backend/routes/signup.js:58]: never mind this is the correct way to do it

**Review by sfeye — APPROVED** [review]
Tested and works

**Review by rlh4g9 — APPROVED** [review]

**Pull request merged**